src/context/ContextProvider.tsx

```diff
@@ -0,0 +1,60 @@
+import React, {createContext, useState} from 'react';
+
+import {fakeData} from '../assets/fake-data';
+import {IToken} from '../interfaces/token.interface';
+
+interface ContextProps {
+    modalInputOpen: boolean;
+    setInputModalOpen: React.Dispatch<React.SetStateAction<boolean>>;
+    modalOutputOpen: boolean;
+    setOutputModalOpen: React.Dispatch<React.SetStateAction<boolean>>;
+    outputToken: IToken | undefined;
+    setOutputToken: React.Dispatch<React.SetStateAction<IToken | undefined>>;
+    inputToken: IToken | undefined;
+    setInputToken: React.Dispatch<React.SetStateAction<IToken | undefined>>;
+}
+
+export const Context = createContext<ContextProps>({
```

Comment:
create a name for this Context, cuz right now it's like a generic name Component or useHook